### DISCUSSION TITLE
helix_view::editor [ERROR] Failed to initialize the LSP for `source.sql` { LSP not defined }

### DISCUSSION THREAD

**Discussion by grv07** [helix-editor/helix#10143] [Troubleshooting]
I try to open a .sql file and it takes around 40-50 sec just to open the editor.

I check my helix logs and there is no information on why/where it takes time.

**Comment by archseer** [comment]
No, that's just a noisy error log that should be hidden away.

What helix version are you running? How did you build it? Did you make sure to build in release mode? Do you have a sample `.sql` file we could debug with? It's hard to debug without details, so [please fill out the bug issue template](https://github.com/helix-editor/helix/blob/master/.github/ISSUE_TEMPLATE/bug_report.yaml).

**Comment by grv07** [comment]
> What helix version are you running? 

helix 22.08.1 (fae9203c)

> How did you build it?

With cargo install.

> Do you have a sample .sql file we could debug with?

It's a dump of 64Mb file.
> What terminal do you use?

Ubuntu with gnome-terminal

**Comment by the-mikedavis** [comment]
This most likely falls under https://github.com/helix-editor/helix/issues/338
